org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/prolog/PrologModel.java

```diff
@@ -0,0 +1,187 @@
+/**
+ *  Copyright (c) 2018 Red Hat, Inc. and others.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v2.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v20.html
+ *
+ *  Contributors:
+ *  Red Hat Inc. - initial API and implementation
+ */
+package org.eclipse.lsp4xml.extensions.prolog;
+
+import java.text.MessageFormat;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.eclipse.lsp4j.CompletionItem;
+import org.eclipse.lsp4j.CompletionItemKind;
+import org.eclipse.lsp4j.InsertTextFormat;
+import org.eclipse.lsp4j.MarkupContent;
+import org.eclipse.lsp4j.MarkupKind;
+import org.eclipse.lsp4j.Range;
+import org.eclipse.lsp4j.TextEdit;
+import org.eclipse.lsp4xml.commons.BadLocationException;
+import org.eclipse.lsp4xml.dom.DOMAttr;
+import org.eclipse.lsp4xml.dom.DOMDocument;
+import org.eclipse.lsp4xml.dom.DOMNode;
+import org.eclipse.lsp4xml.dom.parser.ScannerState;
+import org.eclipse.lsp4xml.dom.parser.TokenType;
+import org.eclipse.lsp4xml.services.AttributeCompletionItem;
+import org.eclipse.lsp4xml.services.XMLCompletions;
+import org.eclipse.lsp4xml.services.extensions.ICompletionRequest;
+import org.eclipse.lsp4xml.services.extensions.ICompletionResponse;
+import org.eclipse.lsp4xml.settings.SharedSettings;
+
+/**
+ * This class holds values that represent the XSI xsd. Can be seen at
+ * https://www.w3.org/2001/XMLSchema-instance
+ */
+public class PrologModel {
+
+	private static final Logger LOGGER = Logger.getLogger(PrologModel.class.getName());
+
+	public static final String VERSION_NAME = "version";
+	public static final String ENCODING_NAME = "encoding";
+
+	public static final String VERSION_1 = "1.0";
+	public static final String VERSION_2 = "2.0";
```

Comment:
There's no such thing as a version 2.0 at that point. https://en.wikipedia.org/wiki/XML#Versions
There's a version 1.1, but it's pretty uncommon